### PR TITLE
Added plugin management. 

### PR DESCRIPTION
**Pull request by washkinazy**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```
Comment (thomasem) [inline Gemfile.lock]: This looks dangerous. Why are we not using TLS?

Comment (thomasem) [inline Gemfile.lock]: This looks dangerous. Why are we not using TLS?

File: elasticsearch/settings.sls [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
-    {% set pkg_version = salt['pkg.version'](elasticsearch_map.pkg) %}
-    {% if pkg_version %}
-      {% set major_version = pkg_version[0] | int %}
-    {% else %}
-      {% set major_version = False %}
-    {% endif %}
-  {% endif %}
-{% endif %}
+{%- set version = salt['pillar.get']('kibana:version', '5.4.3') %}
+{%- set major_version = version[0] | int %}
```
Comment (thomasem) [inline elasticsearch/settings.sls]: Appreciate how much cleaner this is. 😄

Comment (washkinazy) [reply elasticsearch/settings.sls]: Missed that, correcting.

File: Gemfile.lock (hunk 1 above)
Comment (washkinazy) [reply Gemfile.lock]: missed that. corrected.

File: elasticsearch/settings.sls (hunk 2 above)
Comment (washkinazy) [reply elasticsearch/settings.sls]: The original project had it there for good reason. We do not need it so I gutted it.

**Pull request merged**